src/System.Windows.Extensions/tests/System/Drawing/FontConverterTests.cs

```diff
@@ -34,4 +36,119 @@ public void ExTestConvertFrom_ThrowsNotSupportedException()
             Assert.Throws<NotSupportedException>(() => converter.ConvertFrom(1));
         }
     }
+
+    public class FontConverterTest
+    {
+        public static char sep = CultureInfo.CurrentCulture.TextInfo.ListSeparator[0];
+
+        [ConditionalTheory(Helpers.IsDrawingSupported)]
+        [MemberData(nameof(TestConvertFormData))]
+        public void TestConvertFrom(string input, string expectedName, float expectedSize, GraphicsUnit expectedUnits, FontStyle expectedFontStyle)
+        {
+            FontConverter converter = new FontConverter();
+            Font font = (Font)converter.ConvertFrom(input);
+            Assert.Equal(expectedName, font.Name);
+            Assert.Equal(expectedSize, font.Size);
+            Assert.Equal(expectedUnits, font.Unit);
+            Assert.Equal(expectedFontStyle, font.Style);
+        }
+
+        [ConditionalTheory(Helpers.IsDrawingSupported)]
+        [MemberData(nameof(ArgumentExceptionFontConverterData))]
+        public void InvalidInputThrowsArgumentException(string input)
+        {
+            FontConverter converter = new FontConverter();
+            Assert.Throws<ArgumentException>(() => converter.ConvertFrom(input));
```

Comment:
Can you test that parameter name is correct?  I think there is an xunit overload for this.